pkg/resources/tag_association.go

```diff
@@ -18,6 +19,68 @@ import (
 	snowflakeValidation "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/validation"
 )
 
+var (
```

Comment:
why here and not in SDK? (like ValidGrantOwnershipObjectTypesString and the rest?)

Reply:
Moved to a new file in sdk tag_association_validations (similarly to what we have in grants)